spec/lib/resque-serializer/serializers/both_spec.rb

```diff
@@ -21,107 +21,68 @@ def self.perform(*args); end
 end
 
 RSpec.describe JobSerializedByBoth do
-  let(:args) { %w(arg1 arg2) }
+  let(:args)       { %w[arg1 arg2] }
+  let(:queue_name) { :default }
+  let(:worker_1)   { Resque::Worker.new(queue_name) }
+  let(:worker_2)   { Resque::Worker.new(queue_name) }
+
+  subject(:execute_job) { worker_2.reserve }
```

Comment:
Let's turn this into a (shared) helper method, and add a `:enqueue_job` helper; I think it'll make the `before` & `it` blocks a bit more readable.